lib/eco/nodeeditor.py

```diff
@@ -446,6 +448,34 @@ def paint_nodes(self, paint, node, x, y, line, max_y, lbox=0):
 
         return x, y, line
 
+    def draw_lbox_bracket(self, paint, bracket, node, x, y):
```

Comment:
I think we should add a line along the lines of `assert bracket in ["[", "]"]` to avert potential misuses of the `bracket` variable (which otherwise won't be caught and will lead to confusion).